Gemfile

```diff
@@ -1,18 +1,32 @@
 source 'https://rubygems.org'
 
-if puppetversion == ENV['PUPPET_GEM_VERSION']
+if puppetversion = ENV['PUPPET_GEM_VERSION']
```

Comment:
hmmmm, changing it to ```==``` it fails with this:
<pre>
[!] There was an error parsing `Gemfile`: Undefined local variable or method `puppetversion' for Gemfile. Bundler cannot continue.

 #  from /home/ephifre/puppet/puppet-module-puppet-propyless/Gemfile:3
 #  -------------------------------------------
 #
 >  if puppetversion == ENV['PUPPET_GEM_VERSION']
 #    gem 'puppet', puppetversion, :require => false
 #  -------------------------------------------
</pre>